### PR TITLE
Add OAuth and Sign In with Apple to README

### PR DESCRIPTION
**Pull request by morganchen12**

### CODE REVIEW

File: Auth/README.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ FirebaseUI can be easily customized to fit in with the rest of your app's visual
 style, and it is open source, so you aren't constrained in realizing the user
 experience you want.
 
-Compatible FirebaseUI clients are also available for [Android](https://github.com/firebase/firebaseui-android/tree/master/auth)
+Compatible FirebaseUI Auth clients are also available for
+[Android](https://github.com/firebase/firebaseui-android/tree/master/auth)
```
Comment (peterfriese) [inline Auth/README.md]: URL for the web doesn't contain the path to the master branch, can we shorten this one here to https://github.com/firebase/firebaseui-android/ ?

File: Auth/README.md [hunk 2]
```diff
@@ -49,6 +50,8 @@ Auth guides at the following links:
 - [Google](https://firebase.google.com/docs/auth/ios/google-signin#before_you_begin)
 - [Facebook](https://firebase.google.com/docs/auth/ios/facebook-login#before_you_begin)
 - [Phone](https://firebase.google.com/docs/auth/ios/phone-auth#before_you_begin)
+- [Sign in with Apple](https://firebase.google.com/docs/auth/ios/apple#before_you_begin)
+  - For Sign in with Apple, read the *Comply with Apple anonymized data requirements* section as well.
```
Comment (peterfriese) [inline Auth/README.md]: Insert link to https://firebase.google.com/docs/auth/ios/apple#comply-with-apple-anonymized-data-requirements

File: Auth/README.md [hunk 3]
```diff
@@ -109,14 +117,21 @@ NSArray<id<FUIAuthProvider>> *providers = @[
...
+  [FUIOAuth yahooAuthProvider]
 ];
-_authUI.providers = providers;
+self.authUI.providers = providers;
 ```
 
 For Google Sign-in support, add custom URL schemes to your Xcode project
 (step 1 of the [implement Google Sign-In documentation](https://developers.google.com/firebase/docs/auth/ios/google-signin#2_implement_google_sign-in)).
 
+For Sign In with Apple support, add the Sign In with Apple capability to your entitlements file.
```
Comment (peterfriese) [inline Auth/README.md]: Use italics?

Comment (peterfriese) [inline Auth/README.md]: Here as well?

File: Auth/README.md (hunk 1 above)
Comment (morganchen12) [reply Auth/README.md]: The reason these two URLs are different is FirebaseUI Android contains non-Auth components but FirebaseUI Web does not.

File: Auth/README.md (hunk 3 above)
Comment (morganchen12) [reply Auth/README.md]: This formatting matches the Google Sign-in and Facebook login formatting above and below. Sign in with Apple is not usually italicized in Apple's documentation either ([example](https://support.apple.com/en-us/HT210318)).

**Pull request merged**